bot/exts/fun/hangman.py

```diff
@@ -0,0 +1,196 @@
+from asyncio import TimeoutError
+from random import choice
+from typing import Literal
+
+from discord import Embed, Message
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+# Defining all words in the list of words as a global variable
+with open("bot/resources/fun/hangman_words.txt", encoding="utf-8") as f:
+    ALL_WORDS = [line.strip('\n') for line in f.readlines()]
```

Comment:
To be consistent with other files, you can use [`Path.read_text`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text) here. Something like this:
```suggestion
ALL_WORLDS = Path("bot/resources/fun/hangman_words.txt").read_text().splitlines()
```